binaries/scheduler/main.go

```diff
@@ -17,6 +18,11 @@ func main() {
 	flag.Parse()
 
 	parser := config.DefaultParser()
+	parser.Workers[""] = &config.RPCWorkersConfig{}
```

Comment:
If this is going to be the default you should give it a full config. E.g., set Type.

(This means that if you try to marshal it with json, you get a working config)